README.md

```diff
@@ -1,54 +1,78 @@
 
 ## Getting started
 
 ### Requirements
+
+The following must be installed before using Prescient:
+
 * Python 3.7 or later
-* Pyomo 6.1.2 or later
-* EGRET
 * A mixed-integer linear programming (MILP) solver
   * Open source: CBC, GLPK, SCIP, ...
   * Commercial: CPLEX, Gurobi, Xpress, ...
 
 ### Installation
-1. Install EGRET according to the instructions [here](https://github.com/grid-parity-exchange/Egret/blob/main/README.md).
-2. Clone or download/extract the Prescient repository.
-3. The root of the repository will contain a `setup.py` file. In the command prompt or terminal, change your working directory to the root of the repository. Execute the following command:
-```
-python setup.py develop
-```
-This will install Prescient as a Python module while obtaining necessary dependencies. It will also define some new commands in your operating system shell. After installation, your shell should recognize the following command:
+
+#### Installing via pip
+
+Prescient is available as a python package that can be installed using pip. To
+install the latest release of Prescient use the following command:
 
 ```
-D:\workspace\prescient>runner.py
-You must list the file with the program configurations
-after the program name
-Usage: runner.py config_file
+pip install gridx-prescient
 ```
 
-The installation of Prescient has added the `runner.py` command which is used to execute Prescient programs from the command line.
+#### Installing from source
+
+You may want to install from source if you want to use the latest pre-release
+version of the code, or if you want to modify/contribute to the code yourself.
+Install from source by following these steps:
+
+1. Install EGRET from source according to the instructions [here](https://github.com/grid-parity-exchange/Egret/blob/main/README.md).
+   This is necessary because pre-release versions of Prescient sometimes depend 
+   on pre-release versions of EGRET.
+2. Clone or download/extract the Prescient repository.
+3. Install Prescient into the environment as an editable package by issuing the 
+   following command from the root of the Prescient repository:
+   ```
+   pip install -e .
+   ```
+   This will install Prescient as a Python module while obtaining necessary dependencies. 
+
+### Solver Installation
+
+Prescient requires a pyomo-compatible MILP solver. A commercial solver (CPLEX, 
+Gurobi, or Xpress) is recommended over open source solvers. If a commercial 
+solver is not available we recommend installing the open source CBC MILP solver. 
+
+#### CBC
 
-### Solvers
-If not using one of the commerical solvers describe below, we recommend users install the open source CBC MILP solver. The specific mechanics of installing CBC are platform-specific. When using Anaconda on Linux and Mac platforms, this can be accomplished simply by:
+The specific mechanics of installing CBC are platform-specific. When using 
+Anaconda on Linux and Mac platforms, this can be accomplished simply by:
```

Comment:
```suggestion
Anaconda on Windows, Mac, or Linux platforms, this can be accomplished simply by:
```
Cbc is now available via anaconda for Windows: https://anaconda.org/conda-forge/coincbc

Reply:
Thanks, instructions updated...